packages/elements/src/components/ino-radio-group/ino-radio-group.scss

```
@@ -3,6 +3,12 @@ ino-radio-group {
     width: fit-content;
   }
 
+  .ino-radio-group-wrapper-vertical {
+    width: fit-content;
+    display: flex;
+    flex-direction: column;
+  }
+
```

I think I would prefer this way because it's more explicit:

```suggestion
    width: fit-content;
    display: flex;
  
  	&--vertical {
  	    flex-direction: column;
  	}
  	
  	&--horizontal {
  	    flex-direction: row;
  	}
	}

```